class.jetpack.php

```diff
@@ -3253,6 +3251,50 @@ public static function unlink_user( $user_id = null ) {
 		return true;
 	}
 
+	public static function update_disconnection_notice( $user_id ) {
+		$notice = Jetpack_Options::get_option( 'disconnection_notice' );
+		$notice = ! empty( $notice ) && is_array( $notice ) ? array_merge( $notice, array( $user_id ) ) : array( $user_id );
+		Jetpack_Options::update_option( 'disconnection_notice', $notice );
+	}
+
+	/**
+	 * Removes a user's tokens that was already been removed on the .com but not yet on the Jetpack side
+	 * Disconnects the site if the master tokens are deleted
+	 *
+	 * @param $user_id
+	 */
+	public static function unlink_user_on_unknown_token( $user_id ) {
+		$master_user = (int) Jetpack_Options::get_option( 'master_user' );
+		if ( $user_id == 0 || $user_id == JETPACK_MASTER_USER ) {
```

Comment:
This should be a strict operand. It's the reason why it's [not deleting the tokens](https://github.com/Automattic/jetpack/pull/9297#issuecomment-384102252) of the expected user passed to this function. 

```
public static function unlink_user_on_unknown_token( $user_id ) {
	$master_user = (int) Jetpack_Options::get_option( 'master_user' );
	var_dump( "Does $user_id == ". JETPACK_MASTER_USER . " ?" );
	var_dump( $user_id == JETPACK_MASTER_USER );
		return;
```

output: 
```
wp> Jetpack::unlink_user_on_unknown_token( 32 );
string(14) "Does 32 == 1 ?"
bool(true)
```

Reply:
`JETPACK_MASTER_USER` is never a user ID in Jetpack. It's always just `true`. It's only useful as a flag in certain token functions.